bridge/core/bridge.go

```diff
@@ -81,15 +85,33 @@ func NewBridge(repo *cache.RepoCache, target string, name string) (*Bridge, erro
 	return bridge, nil
 }
 
-// Instantiate a new bridge for a repo, from the combined target and name contained
-// in the full name
-func NewBridgeFromFullName(repo *cache.RepoCache, fullName string) (*Bridge, error) {
-	target, name, err := splitFullName(fullName)
+// LoadBridge instantiate a new bridge from a repo configuration
+func LoadBridge(repo *cache.RepoCache, name string) (*Bridge, error) {
+	bridge := &Bridge{
+		Name: name,
+		repo: repo,
+	}
+
+	conf, err := bridge.loadConfig()
 	if err != nil {
 		return nil, err
 	}
+	bridge.conf = conf
+
+	target := bridge.conf[keyTarget]
```

Comment:
That's pretty much the implementation of `NewBridge()`, so why the need to copy it instead of using it ? If you want to change the API to have LoadBridge instead, you should remove NewBridge() or at least deprecate it.

Reply:
The problem i encountered while writing this was that `LoadBridge` need to read the configuration before instantiating a bridge object, and since `loadConfig()` is a method of `Bridge` i was forced to do this.
`NewBridge` is used by `git bug bridge configure` who already knows the target name and want to instantiate a bridge directly. 

What do you think of making `loadConfig` a more general function: something like `loadConfig(repo *cache.Repo, name string)` and then call `NewBridge` from  `LoadBridge` to avoid code repetition ?

Reply:
sounds good to me

Reply:
Done !